pymc3_hmm/utils.py

```diff
@@ -152,7 +152,7 @@ def tt_expand_dims(x, dims):
     dim_range = list(range(x.ndim))
     for d in sorted(np.atleast_1d(dims), reverse=True):
         offset = 0 if d >= 0 else len(dim_range) + 1
-        dim_range.insert(d + offset, "x")
+        dim_range.insert(d + offset, "x")  # type: ignore[arg-type]
```

Comment:
mypy is saying: "Argument 2 to "insert" of "list" has incompatible type "str"; expected "int"  [arg-type]
Found 1 error in 1 file"

@brandonwillard any suggestions on how to avoid this mypy error without using a `#: ignore` comment?